src/utils/lollipop/dependency.ts

```diff
@@ -1,8 +1,11 @@
 import { BlobService } from "azure-storage";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as redis from "redis";
 import { Client } from "../../generated/definitions/fn-lollipop/client";
 
 export type FnLollipopClient = Client<"ApiKeyAuth">;
 export type FnLollipopClientDependency = {
   readonly fnLollipopClient: FnLollipopClient;
   readonly blobService: BlobService;
+  readonly redisClientTask: TE.TaskEither<Error, redis.RedisClientType>;
 };
```

Comment:
```suggestion
export type FnLollipopClientDependency = {
  readonly fnLollipopClient: FnLollipopClient;
  readonly blobService: BlobService;
} & RedisDependency;
```

Reply:
done in https://github.com/pagopa/io-functions-fast-login/pull/27/commits/f31eed4e20119c91d4e2b9977660f7823ed56d1a